File: fib.go
package main

import "fmt"

func fibonacci(n int) int {
    if n < 3 {
        return 1
    }
    return fibonacci(n-1) + fibonacci(n-2)
}

func main() {
    for i := 1; i <= 16; i++ {
        fmt.Print(fibonacci(i), ", ")
    }
    fmt.Println("...")
}

File: fib.java
public class Fibonacci {
    public static int fibonacci(int n) {
        if (n < 3) {
            return 1;
        } else {
            return fibonacci(n - 1) + fibonacci(n - 2);
        }
    }

    public static void main(String[] args) {
        for (int i = 1; i < 17; i++) {
            System.out.print(fibonacci(i) + ", ");
        }
        System.out.println("...");
    }
}
